### PR TITLE
Support Exp

### PR DESCRIPTION
**Pull request by nomaddo**
- Support Exp

### CODE REVIEW

**Review by notogawa — COMMENTED** [review]
plz. add `test_Exp` to [`nnoir/test/test_run.py`](https://github.com/Idein/nnoir/blob/append_exp/nnoir/test/test_run.py)

**Comment by nomaddo** [comment]
> plz. add test_Exp to nnoir/test/test_run.py

Fixed in cb58ef2

File: nnoir/test/test_run.py [hunk 1]
```diff
@@ -456,3 +456,11 @@ def test_Unpooling2D():
         outh=out_h,
         outw=out_w,
     )
+
+
+def test_Exp():
```
Comment (notogawa) [inline nnoir/test/test_run.py:461]: plz move `test_Exp` to between `test_ELU` and `test_Gemm` to keep lexicographic order.

**Review by notogawa — CHANGES_REQUESTED** [review]

**Review by notogawa — APPROVED** [review]
LGTM

**Review by ishiy1993 — APPROVED** [review]
LGTM

**Comment by nomaddo** [comment]
🙏

**Pull request merged**